Description: Update raspberrypi-native.cfg for Dexter Industries BrickPi3

In ev3dev, this is used to update the BrickPi3 firmware.
---

--- openocd-0.10.0.orig/tcl/interface/raspberrypi-native.cfg
+++ openocd-0.10.0/tcl/interface/raspberrypi-native.cfg
@@ -19,7 +19,11 @@ bcm2835gpio_speed_coeffs 113714 28
 
 # Each of the JTAG lines need a gpio number set: tck tms tdi tdo
 # Header pin numbers: 23 22 19 21
-bcm2835gpio_jtag_nums 11 25 10 9
+# bcm2835gpio_jtag_nums 11 25 10 9
+
+# Each of the SWD lines need a gpio number set: swclk swdio
+# Header pin numbers: 22 18
+bcm2835gpio_swd_nums 25 24
 
 # If you define trst or srst, use appropriate reset_config
 # Header pin numbers: TRST - 26, SRST - 18
@@ -27,8 +31,8 @@ bcm2835gpio_jtag_nums 11 25 10 9
 # bcm2835gpio_trst_num 7
 # reset_config trst_only
 
-# bcm2835gpio_srst_num 24
-# reset_config srst_only srst_push_pull
+bcm2835gpio_srst_num 18
+reset_config srst_only srst_push_pull
 
 # or if you have both connected,
 # reset_config trst_and_srst srst_push_pull
